config.c

```diff
@@ -643,7 +643,7 @@ static void set_criterium(enum criterium *pDst, enum criterium src, int *pSet)
 {
     if (*pSet && (*pDst != src)) {
         /* we are overriding a previously set criterium */
-        message(MESS_VERBOSE, "warning: '%s' overrides previously specified '%s'\n",
+        message(MESS_DEBUG, "warning: '%s' overrides previously specified '%s'\n",
```

Comment:
Since `warning: ` is now a common prefix, maybe rename this one to something similar to `note: `?

Reply:
Good idea!  Renamed and force pushed...